CHANGELOG.rst

```diff
@@ -4,10 +4,15 @@ Changelog
 This file contains a brief summary of new features and dependency changes or
 releases, in reverse chronological order.
 
-v2.2.0
+v3.0.0
 ------
 
+* Dropped ``--urgent`` and introduced ``--priority``
 * Added support for filtering by priority
+
+v2.2.0
+------
+
```

Comment:
I meant "remove the v2.2.0" title; the changes should stay since they be released in 3.0.0